### PR TITLE
Add cache in memory for convenience

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

**Comment by ametaireau** [comment]
I actually would like to challenge the usage of in-memory backends. I tend to think this is more code to maintain for something no-one will ever need (appart tests maybe, but running from redis will be even faster).

**Comment by leplatrem** [comment]
I like the idea of frontend developpers, cloning the project and running it locally with a simple ``make serve`` (btw see #124)

You may be right, the solution could be to use https://pypi.python.org/pypi/redislite instead ?

File: cliquet/tests/test_authentication.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def test_views_are_forbidden_if_oauth2_scope_mismatch(self):
 
 class TokenVerificationCacheTest(unittest.TestCase):
     def setUp(self):
-        cache = redis_backend.Redis(max_connections=1)
+        cache = memory_backend.Memory(max_connections=1)
```
Comment (Natim) [inline cliquet/tests/test_authentication.py]: You do not need the ``max_connections`` here

**Comment by ametaireau** [comment]
That sounds less code to maintain at least!

**Comment by ametaireau** [comment]
Are we okay to close this?

**Comment by Natim** [comment]
I'd rather merge it.

**Comment by Natim** [comment]
Lets merge it.

**Pull request merged**